red-hat-data-services/odh-deployer
# Fix logic for ISV Deployment

## Description

Fix deployment for ISV apps.

## How Has This Been Tested?



1. Install the operator on a managed cloud
2. Check that starbust enterprise is not installed

<img width="1083" alt="Screenshot 2023-06-08 at 14 04 23" src="https://github.com/red-hat-data-services/odh-deployer/assets/16117276/357575d7-a87c-4105-915b-6e1098e48827">

## Merge criteria:



- [X] JIRA link(s): https://issues.redhat.com/browse/RHODS-9226
- [X] Live build image: `quay.io/lferrnan/rhods-operator-live-catalog:1.28.0-deployer`
